src/app/content/components/Toolbar/styled.tsx

```diff
@@ -114,22 +114,25 @@ export const PrintOptions = styled.span`
 export const PrintIcon = styled(Print)`
   ${toolbarIconStyles}
 `;
+
 // tslint:disable-next-line:variable-name
-export const SearchButton = styled(({ desktop, mobile, ...props }) => <PlainButton {...props}><Search /></PlainButton>)`
-  > svg {
-    ${toolbarIconStyles}
-  }
+export const SearchButton = styled(({ desktop, mobile, ...props }) =>
+  <PlainButton {...props}><SearchIcon/></PlainButton>)`
```

Comment:
formatting needs work... put a newline after the open `styled(` and put the closing `)`` on a new line

Reply:
SearchIcon is an asset, in other places you're putting that in the src of images, here you're just rendering it. why the difference? does this work?

Reply:
https://github.com/openstax/rex-web/pull/955#discussion_r536139234